docs/apache-airflow/templates-ref.rst

```diff
@@ -79,8 +79,6 @@ Variable                                    Type                  Description
 ``{{ conn }}``                                                    Airflow connections. See `Airflow Connections in Templates`_ below.
 ``{{ task_instance_key_str }}``             str                   | A unique, human-readable key to the task instance. The format is
                                                                   | ``{dag_id}__{task_id}__{ds_nodash}``.
-``{{ conf }}``                              AirflowConfigParser   | The full configuration object representing the content of your
```

Comment:
Is there a partner PR such that `conf` in the context is added to the deprecated fields?

Reply:
+1 We need to add a deprecation PR for this.

Reply:
Yup, absolutely, I will have one I promise :) -- I am now waiting for the lazy consensus to end and will create one to have a deprecation PR

Reply:
Created https://github.com/apache/airflow/pull/44968